.circleci/config.yml

```diff
@@ -247,7 +247,7 @@ jobs:
     macos:
       xcode: 11.7.0
     environment:
-      BUILDIMAGES_VERSION: "0f65dfe0488a16bcf63b19e0376185bfcac74503" # datadog-agent-buildimages commit to fetch to get the MacOS scripts
+      BUILDIMAGES_VERSION: "249dd7120e29cf04436a116834f5a3e7f634b6a7" # datadog-agent-buildimages commit to fetch to get the MacOS scripts
```

Comment:
You probably also want to bump the cache number in the `save_cache` and `restore_cache` sections below (macOS cache is specially fragile because `brew` is not really built with caching in mind)

Reply:
Nice, thanks!